reana_commons/openapi_specifications/reana_server.json

```diff
@@ -355,7 +355,8 @@
                   "value": [
                     "kubernetes",
                     "htcondorcern",
-                    "slurmcern"
+                    "slurmcern",
+                    "compute4punch"
```

Comment:
I will open the related reana-server PR

Reply:
This change is actually not needed, given that the code changed is only an example. Maybe we can avoid changing this altogether?